cli/cmd/plugin/ui/web/tanzu-ui/src/components/aws/AwsWizard.tsx

```diff
@@ -0,0 +1,25 @@
+// React imports
```

Comment:
I am thinking that we should name this high level component specific to the type of cluster it is used for creating. So something like AwsManagementClusterWizard.tsx

We may also want to consider updating the directory structure. Let's chat over zoom.